sale_order_line_position/models/position_mixin.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class OrderPositionMixin(models.AbstractModel):
+    _name = "order.position.mixin"
+    _description = "Order position mixin"
```

Comment:
You can add a class attribute like `position_field_name`, which would have to be set as `order_lines` on `sale.order` and so on.
By doing that, you just have to define this attribute in order to set that up, instead of having to define methods in classes extending this mixin.

Reply:
you can take this as an example:
https://github.com/OCA/stock-logistics-warehouse/blob/14.0/stock_packaging_calculator/models/product_qty_by_packaging_mixin.py#L14